Add helper to sync maintainers file

jepio opened the pull request.

Commit:
Add helper to sync maintainers file

Correct MAINTAINERS.md entry to match repo names.

sayanchowdhury commented on sync-maintainers/sync-maintainers.py:
```diff
@@ -0,0 +1,148 @@
+def write_maintainers_file(repo_name, paragraph, maintainers):
+    maintainers_entry = '\n'.join(maintainers)
+    maintainers_content = MAINTAINERS_TEMPLATE.format(maintainers=maintainers_entry, paragraph=paragraph)
+    repo_filename = '{}/MAINTAINERS.md'.format(repo_name)
```
```suggestion
    repo_filename = f'{repo_name}/MAINTAINERS.md'
```

jepio replied:
replaced all format calls (except one) with f-strings

sayanchowdhury commented on sync-maintainers/sync-maintainers.py:
```diff
@@ -0,0 +1,148 @@
+            print('{} creating pr'.format(repo_name))
+            base = get_default_branch(repo_name, token)
+            pr = create_pr(repo_name, token, base).json()
+            pr = [pr]
```
```suggestion
            pr = [create_pr(repo_name, token, base).json()]
```

sayanchowdhury commented on sync-maintainers/sync-maintainers.py:
```diff
@@ -0,0 +1,148 @@
+if __name__ == '__main__':
+    args = parser.parse_args()
+    args.func(args)
+    #main()
```
```suggestion
```

sayanchowdhury reviewed (commented):
Looks good. All the `.format` could be changed to f-strings and would be great if you can run [black](https://pypi.org/project/black/) to format the code.

Commit:
sync-maintainers: Address review comments

Use f-strings, remove commented line, combine two statements into one line, add
comment clarifying token scope to README, and reformat with black.

krnowak commented on sync-maintainers/sync-maintainers.py:
```diff
@@ -0,0 +1,190 @@
+
+{paragraph}
+
+The contents of this file are synchronized from [Flatcar/MAINTAINERS.md](https://github.com/flatcar-linux/Flatcar/blob/main/MAINTAINERS.md).
```
```suggestion
The contents of this file are synchronized from [Flatcar/MAINTAINERS.md](https://github.com/flatcar/Flatcar/blob/main/MAINTAINERS.md).
```
